### PR TITLE
Release q3 2022

### PR DESCRIPTION
**Pull request by apaparazzi0329**
Assistant V1
### Models
* RuntimeIntent: `confidence` is no longer a required parameter
* Workspace: New status available: “processing”, new properties added: `statusErrors` and `counts`
* WorkspaceSystemSettings: New property `nlp`

### Methods
* NEW Method: `createWorkspaceAsync`
* NEW Method: `updateWorkspaceAsync`
* NEW Method: `exportWorkspaceAsync`

### Models
* BulkClassifyOptions: add `input` as required property
* MessageOutputDebug: add `turnEvents` property
* RuntimeEntity: add `skill` property
* RuntimeIntent: add `skill` property and `confidence` no longer a required property
* RuntimeResponseGeneric: Date type added

### Methods
* NEW Method: `listEnvironments`
* NEW Method: `getEnvironments`
* NEW Method: `listReleases`
* NEW Method: `getRelease`
* NEW Method: `deployRelease`

## Discovery V1
* No changes

### Models
* add parameter  `smartDocumentUnderstanding` to `createCollection`  and `updateCollection`
* Enrichment: new type `clarifier`
* add parameters `classifierId`, `modelId`, `confidenceThreshold`, `topK` to `createEnrichment` and `updateEnrichment`
* add parameter `similar` to `query`

### Methods
* NEW Method: `listDocuments`
* NEW Method: `getDocument`
* NEW Method: `listDocumentClassifiers`
* NEW Method: `createDocumentClassifier`
* NEW Method: `getDocumentClassifier`
* NEW Method: `updateDocumentClassifier`
* NEW Method: `deleteDocumentClassifier`
* NEW Method: `listDocumentClassifierModels`
* NEW Method: `createDocumentClassifierModel`
* NEW Method: `getDocumentClassifierModel`
* NEW Method: `updateDocumentClassifierModel`
* NEW Method: `deleteDocumentClassifierModel`
* NEW Method: `getStopwordList`
* NEW Method: `createStopwordList`
* NEW Method: `deleteStopwordList`
* NEW Method: `listExpansions`
* NEW Method: `createExpansions`
* NEW Method: `deleteExpansions`

## LT
* Nothing

### Models
* add parameter `trainingParameters` to `createClassificationsModel` and `updateClassificationsModel`

### Models
* Remove parameter `customizationId`  from  `createJob` and `recognize`
* Add parameter `characterInsertionBias` to `createJob` and `recognize`
* Add parameter `strict` to `trainAcousticModel` and `trainLanguageModel`

### Models
* Add parameter  `spellOutMode` to `synthesize`

### CODE REVIEW

**Review by kevinkowa — CHANGES_REQUESTED** [review]
Make sure you run detect-secrets

**Comment by apaparazzi0329** [comment]
Done

File: assistant/src/main/java/com/ibm/watson/assistant/v1/model/DialogNodeOutputGenericDialogNodeOutputResponseTypeAudio.java [hunk 1]
```diff
@@ -157,6 +157,8 @@ public Builder altText(String altText) {
     }
   }
 
+  protected DialogNodeOutputGenericDialogNodeOutputResponseTypeAudio() {}
```
Comment (kevinkowa) [inline assistant/src/main/java/com/ibm/watson/assistant/v1/model/DialogNodeOutputGenericDialogNodeOutputResponseTypeAudio.java:160]: I see that this type of empty protected class is happening everywhere, I think that's part of the new version for the generator

**Review by kevinkowa — APPROVED** [review]
Looks good to me 👍 🥇

**Review by kevinkowa — APPROVED** [review]
LGTM!

**Pull request merged**